src/__integ__/customEvents.test.ts

```diff
@@ -127,6 +127,13 @@ test('when a plugin calls recordEvent then the event is recorded', async (t: Tes
 });
 
 test('when a plugin calls recordEvent x times then event is recorded x times', async (t: TestController) => {
+    // This test passes individually, but fails on Edge when run as apart of the suite
```

Comment:
These tests succeed individually, but fail when run with the rest of the suite. It is unclear how the changes in this PR are related to this behavior change, or if it's an issue with TestCafe.